cityofaustin/atd-moped
# 17450 explore replacing react quill

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/17450

## Testing
**URL to test:** 

https://deploy-preview-1377--atd-moped-main.netlify.app/

**Steps to test:**
- go to a project, ideally one you commented on previously when we were using quill
- copy and paste a comment with rich text and links
- save the comment, it should appear in the comments section and the editor should clear out and refocus
- click edit, the editor should appear below the comment populated with the selected comment
- edit the comment and save
- try editing an old comment made using quill to confirm it loads in the new editor

this is a proof of concept at the moment, there are a few kinks to work out and features to add (resizing the editor to fit longer comments, adding a toolbar, a few other rich text and link functionalities). those will be addressed in a followup PR!


---
#### Ship list
~- [ ] Product manager added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable~
